src/utils/helper.js

```diff
@@ -47,3 +47,15 @@ export const debounce = (fn, delay) => {
     }, delay);
   };
 };
+
+export const getURLParameters = url => {
+  const keyVal = {};
+  url.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m, key, value) {
+    key = decodeURIComponent(key);
+    value = decodeURIComponent(value);
+    keyVal[key] = value;
+  });
+  return keyVal;
+};
+
+export const isEmpty = val => val == null || !(Object.keys(val) || val).length;
```

Comment:
`val == null` will be true for `undefined` as well. 

I see this is been used at 2 places and the argument passed is of type `string` . So the later condition that is  `keys` and `length` is just for future use ?

Reply:
> val == null will be true for undefined as well.

should return `true` for `undefined`

`(Object.keys(val)` => future
`!val.length` => this is for string, return true for `''`

Reply:
We can simplify this: 

```
!(val && val.length && Object.keys(val).length)
```

Wdyt ?

Reply:
cool